modules/Sources/Utils/NumberFormatter+zcash.swift

```diff
@@ -8,6 +8,14 @@
 import Foundation
 
 extension NumberFormatter {
+    public static let currencyFormatter: NumberFormatter = {
+        var formatter = NumberFormatter()
+        formatter.maximumFractionDigits = 2
+        formatter.numberStyle = .decimal
```

Comment:
[Non-Blocking] I wonder if it's better to use `.currency` instead.

Reply:
.currency adds the symbol, such as '$' or '€', but with our designs we want something like "around 2.50 USD", so that's why I chose this approach.